### PR TITLE
feat(deploy-service): Rematch deploy controller check status, deploy status, deploy cluster status.

### PR DESCRIPTION
**Pull request by Luckyboys**
Making deploy controller operation status convert to deploy service check and deploy status.

### CODE REVIEW

File: pkg/constant/check_result.go [hunk 1]
```diff
@@ -19,8 +19,9 @@ type (
...
 const (
-	CheckResultPending    CheckResult = "pending"
-	CheckResultRunning    CheckResult = "running"
-	CheckResultSuccessful CheckResult = "successful"
-	CheckResultFailed     CheckResult = "failed"
+	CheckResultPending              CheckResult = "pending"
+	CheckResultRunning              CheckResult = "running"
+	CheckResultSuccessful           CheckResult = "successful"
+	CheckResultFailed               CheckResult = "failed"
+	CheckResultDeployServiceUnknown CheckResult = "unknown(deploy)"
```
Comment (randmonkey) [inline pkg/constant/check_result.go]: `unknown(deploy)`是个什么状态

File: pkg/service/api/v1/deploy/convert.go [hunk 2]
```diff
@@ -236,42 +236,50 @@ func convertModelConnectionDataToDeployControllerSSHData(data *wizard.Connection
...
-	case string(wizard.DeployClusterStatusSuccessful):
+	case string(constant.OperationStatusSuccessful):
 		return wizard.DeployClusterStatusSuccessful
-	case string(wizard.DeployClusterStatusFailed):
+	case string(constant.OperationStatusFailed):
 		return wizard.DeployClusterStatusFailed
-	case string(wizard.DeployClusterStatusWorkedButHaveError):
-		return wizard.DeployClusterStatusWorkedButHaveError
+	case string(constant.OperationStatusUnknown):
+		return wizard.DeployClusterStatusDeployServiceUnknown
```
Comment (randmonkey) [inline pkg/service/api/v1/deploy/convert.go]: 为啥`DeployClusterStatusWorkedButHaveError` 改成了`DeployClusterStatusDeployServiceUnknown`

Comment (Luckyboys) [reply pkg/service/api/v1/deploy/convert.go]: deploy controller的Operation那边没有这个状态，需要Service这边进行判断。所以原本的状态加入了一个判断方法进行处理了

File: pkg/constant/check_result.go (hunk 1 above)
Comment (Luckyboys) [reply pkg/constant/check_result.go]: 就是deploy controller那边返回了unknown

**Pull request merged**